packages/calypso-products/src/main.ts

```diff
@@ -481,6 +494,8 @@ export function getBillingMonthsForTerm( term: string ): number {
 		return 12;
 	} else if ( term === TERM_BIENNIALLY ) {
 		return 24;
+	} else if ( term === TERM_TRIENNIALLY ) {
+		return 36;
```

Comment:
Can you also add a triennial case for `getBillingYearsForTerm` and `getBillingTermForMonths`?